pkgs/applications/science/logic/drat-trim/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation {
   pname = "drat-trim-unstable";
-  version = "2020-06-05";
+  version = "2023-05-22";
```

Comment:
```suggestion
  version = "unstable-2023-05-22";
```

Reply:
The package is actually stable. I renamed it to just `drat-trim` and kept the current version.

Reply:
Hmm, I see that upstream has tagged a release but named it `05.22.2023`, which is not lexicographically sortable in comparison to other versions of the same format (e.g. if say `03.15.2024` is released next year, it will be read as older than `05.22.2023`).

Do we want to systematically rewrite such upstream version numbers into the `yyyy-mm-dd` format that we were already using for untagged releases, from now on? (Assuming that upstream continues to use their new format.)